Bottom: a5f29d27203d544232fcd0fb9c185d4b936d8eb7
Top:    ae477bbb01301e38b94c639c4fef3d2067c8df37
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 12:22:59 +0000

TODO notes


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 443a55a..1be059a 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1,3 +1,12 @@
+/* TODO ITEMS:
+ *
+ * Add PLINE simplification operation to consolidate co-circular segments for reduced geometry output.
+ * Look at whether arc-* intersections can be re-constructed back to original geometry, not fall back to line-line.
+ * Work on snap-rounding any edge which passes through the pixel square containing an any vertex (or intersection).
+ * Avoid self-touching output in contours, where that self-touching instance creates two otherwise distinct contours or holes.
+ *
+ */
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
